src/scripts/domParsers/domUtils.ts

```diff
@@ -974,4 +996,13 @@ export module DomUtils {
 
 		return scrollValue * 100;
 	}
+
+	private static isLocalReferenceUrl(url: string): Boolean {
+		return (
+			url.indexOf("chrome-extension://") === 0 ||
```

Comment:
I wonder if we're better off returning false if the url doesn't start with http:// or https://. Also you can loop through the schemes to avoid dupe code.